src/Makefile

```diff
@@ -69,8 +69,8 @@ clean:
 
 %.cpp.d: %.cpp
 	@echo '[DPP] $^'
-	${co}$(CPP) $(BOOST_CPPFLAGS) $(SAL_CPPFLAGS) $(M1M3_CPPFLAGS) -M $^ -MF $@ -MT '$*.o'
+	${co}$(CPP) $(BOOST_CPPFLAGS) $(SAL_CPPFLAGS) $(M1M3_CPPFLAGS) -M $^ -MF $@ -MT '$*.o $*.d'
```

Comment:
What are the `.d` files?

Reply:
.d are for dependencies. Those Makefile-like files listing header files on which the source file depends (through any dependency levels, e.g. directly included, included in a file included, ..). Those are used to trigger recompilation if needed. The change above adds .d file itself to the list, to trigger new dep generation if a header change. This is to trigger compilation when a file is deleted/renamed, which was problem discovered with BootCommand.